ui/main/src/app/modules/share/light-card/light-card.component.ts

```diff
@@ -113,11 +122,25 @@ export class LightCardComponent implements OnInit, OnDestroy {
         }
     }
 
+    private computeGroupedCardsIcon() {
+        this.showGroupedCardsIcon = this.groupedCardsService.isParentGroupCard(this.lightCard);
```

Comment:
to be call only if group option is activated to avoid unnecessary processing

Reply:
Same as above. If the group option is not activated, "isParentGroupCard" calls indexOf on an empty array, which will not use a lot of processing power.

Reply:
ok , you are right